integration-tests/suites/repeated_network_flow.go

```diff
@@ -88,7 +88,28 @@ func (s *RepeatedNetworkFlowTestSuite) SetupSuite() {
 	numIter := strconv.Itoa(s.NumIter)
 	sleepBetweenCurlTime := strconv.Itoa(s.SleepBetweenCurlTime)
 	sleepBetweenIterations := strconv.Itoa(s.SleepBetweenIterations)
-	_, err = s.execContainer("nginx-curl", []string{"/usr/bin/schedule-curls.sh", numMetaIter, numIter, sleepBetweenCurlTime, sleepBetweenIterations, serverAddress})
+	_, err = s.execContainerShellScript("nginx-curl", `
+		#!/usr/bin/env sh
+
+		num_meta_iter=$1
+		num_iter=$2
+		sleep_between_curl_time=$3
+		sleep_between_iterations=$4
+		url=$5
+
+		i=0
+		j=0
+
+		while [ "$i" -lt "$num_meta_iter" ]; do
+		    while [ "$j" -lt "$num_iter" ]; do
+		        curl "$url"
+		        sleep "$sleep_between_curl_time"
+		        j=$((j + 1))
+		    done
+		    sleep "$sleep_between_iterations"
+		    i=$((i + 1))
+		done`,
```

Comment:
A blank line here would be good so that it is easier to see where the script ends.

Reply:
Inserted a newline